x/service/module/tx_add_service.go

```diff
@@ -1,6 +1,7 @@
 package service
```

Comment:
Make sure to review this doc to make sure nothing is missed: https://dev.poktroll.com/develop/developer_guide/adding_params

Reply:
@Olshansk I did review that document, but I can't see anything that I need to update. There are no docs about services within applications for me to edit and there are no make scripts for updating services either. If I missed them, please let me know.

Reply:
I didn't do an exhaustive comparison of everything in the document and this PR but here is just one example:

![Screenshot 2024-07-19 at 4 18 46 PM](https://github.com/user-attachments/assets/4904beae-b23e-4d60-82f8-781a8351119a)

**Required changes:**
1. Add a makefile target that executes an update transaction to update the param you introduced

**Why?**
1. Serve as an example for others
2. This is something we'll need VERY SHORTLY as we start testing it
3. As the person who implemented this feature, it'll be easier/faster for you than whoever picks it up from here
4. It's another way to validate in a real environment (i.e. LocalNet) that the CLI works in conjunction with the real environment.

**Does that make sense?**

I can look through the rest of the document and see if there's anything else.